sldasm_diff.sh

```diff
@@ -14,7 +10,9 @@ fi
 diff_filename="$(echo "$1" | sed 's/\(.*\)\.\([^.]*\)/\1_OTHER\.\2/')"
 cp $2 $diff_filename
 
```

Comment:
```suggestion

echo "Opening SolidWorks to diff $left_filename and $right_filename..."
echo "SolidWorks must exit fully before this diff can exit"
```
it would be nice to tell the user what's happening here because SolidWorks can be very slow to open and they might Ctrl-c or something :sweat_smile:

Reply:
edited to match new commits in `master`